gossip/src/push_active_set.rs

```diff
@@ -110,14 +111,20 @@ impl PushActiveSetEntry {
 
     fn get_nodes<'a>(
         &'a self,
-        origin: &'a Pubkey,
+        pubkey: &'a Pubkey, // This node.
+        origin: &'a Pubkey, // CRDS value owner.
         // If true forces gossip push even if the node has pruned the origin.
         mut should_force_push: impl FnMut(&Pubkey) -> bool + 'a,
     ) -> impl Iterator<Item = &Pubkey> + 'a {
+        let pubkey_eq_origin = pubkey == origin;
         self.0
             .iter()
             .filter(move |(node, bloom_filter)| {
-                !bloom_filter.contains(origin) || should_force_push(node)
+                // Bloom filter can return false positive for origin == pubkey
+                // but a node should always be able to push its own values.
```

Comment:
Might be nice to put this comment right next to the line it applies to (i.e. move it down to line 125.5)

Reply:
I have seen `cargo fmt` not being able to handle intermediate comments with logical operators.
This is already merged by automerge anyways, but nonetheless, since this is a backport, in order to keep master and v2.0 consistent we need to first make such changes on master.